shell/platform/darwin/ios/framework/Source/FlutterEmbedderKeyResponder.mm

```diff
@@ -632,9 +623,9 @@ - (void)sendPrimaryFlutterEvent:(const FlutterKeyEvent&)event
   _responseId += 1;
   uint64_t responseId = _responseId;
   FlutterKeyPendingResponse* pending =
-      [[[FlutterKeyPendingResponse alloc] initWithHandler:self responseId:responseId] autorelease];
+      [[FlutterKeyPendingResponse alloc] initWithHandler:self responseId:responseId];
   [callback pendTo:_pendingResponses withId:responseId];
-  _sendEvent(event, HandleResponse, pending);
+  _sendEvent(event, HandleResponse, (__bridge_retained void* _Nullable)pending);
```

Comment:
We had tests that crashed when this was just `__bridge` (reminder about what this does: https://medium.com/@hadhi631/when-to-use-bridge-bridge-transfer-cfbridgingrelease-and-bridge-retained-cfbridgingretain-4b3d2fc932df)